index.html

```diff
@@ -7,9 +7,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Квест по выставке "Очарования Делакруа"</title>
     <link rel="stylesheet" href="./pages/index.css">
+    <script src="js/index.js"></script>
 </head>
 
-<body class="page">
+<body class="page" onload="renderOnLoadFromLocalStorage()">
```

Comment:
This defines a function which will be called every time webpage is loaded (we need this to re-render the state of user's responses e.g. if our user accidentally reloads the page during the quest).